### DISCUSSION TITLE
Issues Uploading Images to GitHub Pages

### DISCUSSION THREAD

**Discussion by HendricoMaximus** [community/community#22181] [Other Feature Feedback, Questions, & Ideas]
<p>Hey there! Yesterday, I posted asking about adding new pages to a site, but today I’m having new issues adding images. I’ve seen many sites saying to use Issues, strange coincidence, but I don’t even have that.</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/8/e824c74f1041379cb89bb3813367d47225922723.png" data-download-href="/uploads/short-url/x7Dzofswb1MwZORJpQOsELn6QsH.png?dl=1" title="Screen Shot 2020-12-16 at 3.33.27 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75211263/181069375-b016ae8c-1842-46a3-a365-5643736c9588.png" alt="Screen Shot 2020-12-16 at 3.33.27 PM" data-base62-sha1="x7Dzofswb1MwZORJpQOsELn6QsH" width="690" height="43" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e824c74f1041379cb89bb3813367d47225922723_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-12-16 at 3.33.27 PM</span><span class="informations">886×56 4.01 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
Anyway, I thought I had found a way around it, by uploading all files into a folder in the directory and copying the link.
<pre><code class="lang-auto">![Simplistic](https://github.com/HendricoMaximus/hendricomaximus.github.io/blob/e2b392cd65618c1334cfdaa24fb365f8e7db68e7/files/Simplistic.png)```
That's one of the images I'm trying to add, which shows up in the preview, but visiting https://hendricomaximus.github.io/portfolio/ui shows the text and then an image icon. Any idea what's going on? Any help is appreciated!</code></pre>

**Comment by logankilpatrick** [comment, marked as answer]
<p>If you go to the repo settings, you should be able to enable issues. Let me know if you don’t see that option.</p>

**Comment by HendricoMaximus** [comment]
<p>Thank you! I’ll check in a bit. I was wondering if issues was deprecated.</p>

**Comment by logankilpatrick** [comment]
<p>They are still very much a core aspect of GitHub</p>

**Comment by HendricoMaximus** [comment]
<p>Ah. Got it. And it is working now! Thank you so much for your help! Also with the issue yesterday. I  really appreciate it!</p>

**Comment by logankilpatrick** [comment]
<p>Glad you got it working :tada:</p>
